frontend/src/components/Header.jsx

```diff
@@ -20,7 +20,9 @@ export default function Header({ searchHandler }) {
   return (
     <nav className="header">
       <div className="logo-container">
-        <img src={gatsbyLogo} alt="Gatsby Logo" className="logo" />
+        <Link to="http://localhost:3000/">
```

Comment:
the value in the 'to' attribute should be just "/" otherwise this code will break in production because when we deploy the url is no longer "localhost"

Always use the extensions from "/" onwards and react router will fill in the rest.